audit/projects/k8s-artifacts-prod/iam.json

```diff
@@ -29,8 +29,7 @@
       "members": [
         "serviceAccount:388270116193-compute@developer.gserviceaccount.com",
         "serviceAccount:388270116193@cloudservices.gserviceaccount.com",
-        "serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com",
-        "user:justinsb@google.com"
+        "serviceAccount:service-388270116193@containerregistry.iam.gserviceaccount.com"
```

Comment:
This was me making a manual edit.  @justinsb is a member of k8s-infra-artifact-admins so this binding should have been redundant